pkg/credentialwatcher/secretwatcher.go

```diff
@@ -139,20 +139,24 @@ func (s *secretWatcher) updateAccountRotateCredentialsStatus(log logr.Logger, ac
 		return
 	}
 
-	if accountInstance.Status.RotateCredentials != true {
```

Comment:
If I'm understanding the previous code correctly, there was a hole where, if the CLI creds had recently been rotated, we would miss rotating the console creds completely. And this code change makes it so we're checking both of them explicitly. Do I have that right?

Reply:
We now check both explicitly but we also do not update the Account CR's status sub resource if its already the desired value `true` since this would cause an increment in resource version and an error in the reconciler.